src/components/login/google-login/google-login.jsx

```diff
@@ -14,12 +19,12 @@ const GoogleLogin = ({
         render={renderProps => (
           <button
             type="button"
-            className="btnGoogle"
+            className="btn-google"
             onClick={renderProps.onClick}
             disabled={renderProps.disabled}
           >
-            <i className="fab fa-google blue fa-lg" />
-            <span>Log in</span>
+            <FcGoogle className="icon" />
```

Comment:
Please rename this className according to BEM methodology, it should not be just `icon`. If we add more icons, there will be className conflict

Reply:
Ok, changed.